Services/OperationService/Controllers/OperationsController.cs

```diff
@@ -106,6 +105,47 @@ public OperationsController([FromServices] ILogger<OperationsController> logger)
                     Token = token,
                     Figi = figi
                 });
+        }
+
+		[Route("bot/create")]
```

Comment:
We can create new controller specially for operations with bots. This is just a suggestion.

Reply:
I don't think so